svf-llvm/include/SVF-LLVM/LLVMUtil.h

```diff
@@ -100,9 +100,11 @@ static inline Type* getPtrElementType(const PointerType* pty)
 {
 #if (LLVM_VERSION_MAJOR < 14)
     return pty->getPointerElementType();
-#else
+#elif (LLVM_VERSION_MAJOR < 17)
```

Comment:
I remember it is version 16?

Reply:
16 still supports typed pointers (https://llvm.org/docs/OpaquePointers.html#version-support)